### PR TITLE
Mime url prefix v2

### PR DESCRIPTION
**Pull request by regit**
Update of #7103.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
- Fix url logging when no config is set (events are unchanged)
- Rebased on latest master

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 7915

File: src/output-json-email-common.c [hunk 1]
```diff
@@ -289,17 +290,31 @@ static bool EveEmailLogJsonData(const Flow *f, void *state, void *vtx, uint64_t
         int url_cnt = 0;
         JsonBuilder *js_attch = jb_new_array();
         JsonBuilder *js_url = jb_new_array();
+        bool has_ipv6_url = false;
```
Comment (victorjulien) [inline src/output-json-email-common.c:293]: scope can be reduced a level

File: src/output-json-email-common.c [hunk 2]
```diff
@@ -51,6 +51,7 @@
 
 #include "output-json.h"
 #include "output-json-email-common.h"
+#include <stdbool.h>
```
Comment (victorjulien) [inline src/output-json-email-common.c:54]: remove

**Comment by victorjulien** [comment]
Needs a schema update as well. @inashivb could you implement these minor updates and provide a public SV test? I've confirmed the issue by chance in a private test.

**Pull request closed without merging**